scripts/common.psm1

```diff
@@ -16,23 +16,31 @@ function Invoke-Block([scriptblock]$cmd) {
     }
 }
 
-function Get-Submodules([string]$ModuleDirectory)
-{
-    Invoke-Block { & git submodule update --init }
+function Get-Submodules {
+    param(
+        [Parameter(Mandatory = $true)]
+        [string]$RepoRoot
+    )
 
-    $gitModules = Join-Path $RepoRoot ".gitmodules"
+    if ((Get-ChildItem "$RepoRoot/modules/*" -Directory | Measure-Object) -eq 0) {
```

Comment:
So for each subdirectory in `$RepoRoot/modules` which is empty  we update it?

Reply:
actually, tested this more and doesn't work. `git submodules update --init` takes 30-40 seconds. I'm trying to skip it if necessary. I'm going to remove this though as it's possible to have only some git submodules checked out